changelog.txt

```diff
@@ -1,3 +1,9 @@
+4.21.1
+============
+
+Bug fixes: 
+** Fix the URI for IMDS call to detect region **. This fixes the typo in URI for local IMDS call which is made to detect the region for regional auth.
```

Comment:
```suggestion
**Fix the URI for IMDS call to detect the region**. This fixes the typo in the URI for local IMDS call which is made to detect the region for regional auth.
```

Reply:
is there an issue you can link to?

Reply:
Agree would be great with an issue that describes what the issue is, which then links to the PR. It makes things add up meaningfully for those later trying to understand what happened and why.

---
In reply to: [508054053](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2118#discussion_r508054053) [](ancestors = 508054053)